CONTRIBUTING.md

```diff
@@ -1,269 +1,79 @@
-
-### References
-
-The following blog post has a good story how changes in naming of commit messages,
-help maintain the project:
-
-"How to Write a Git Commit Message": <https://chris.beams.io/posts/git-commit/>
-
-## Squashing Commits
-
-If your pull request is accepted for merging, you may be asked by a maintainer
-to squash and or [rebase](https://git-scm.com/docs/git-rebase) your commits
-before it will be merged. The basic squashing workflow is shown below.
-
-```sh
-git checkout your_branch_name
-git rebase -i HEAD~n
-# n is normally the number of commits in the pull
-# set commits from 'pick' to 'squash', save and quit
-# on the next screen, edit/refine commit messages
-# save and quit
-git push -f # (force push to GitHub)
-```
-
-Please refrain from creating several pull requests for the same change.
-Use the pull request that is already open (or was created earlier) to amend changes.
-This preserves the discussion and review that happened earlier
-for the respective change set.
-
-The length of time required for peer review is unpredictable and will vary
-from pull request to pull request.
-
-## Pull Request Philosophy
-
-Patchsets should always be focused. For example, a pull request could add a feature,
-fix a bug, or refactor code; but not a mixture.
-Please also avoid super pull requests which attempt to do too much,
-are overly large, or overly complex as this makes review difficult.
-
-### Features
-
-When adding a new feature, thought must be given to the long term technical debt
-and maintenance that feature may require after inclusion.
-Before proposing a new feature that will require maintenance,
-please consider if you are willing to maintain it (including bug fixing).
-If features get orphaned with no maintainer in the future, they may be removed
-by the Repository Maintainer.
-
-### Refactoring
-
-Refactoring is a necessary part of any software project's evolution.
-The following guidelines cover refactoring pull requests for the project.
-
-There are three categories of refactoring, code only moves,
-code style fixes, code refactoring.
-In general refactoring pull requests should not mix these three kinds
-of activity in order to make refactoring pull requests easy to
-review and uncontroversial. In all cases, refactoring PRs must not change
-the behaviour of code within the pull request (bugs must be preserved as is).
-
-Project maintainers aim for a quick turnaround on refactoring pull requests,
-so where possible keep them short, uncomplex and easy to verify.
-
-## "Decision Making" Process
-
-The following applies to code changes to the Exonum Core project and related
-projects such as Exonum BTC Anchoring Service.
+## Finding something to fix or improve
 
-Whether a pull request is merged into Exonum Core rests with the project merge
-maintainers and ultimately the project lead.
+The [good first issue] label can be used to find the easy issues.
```

Comment:
I'm not quite sure we should advertise this label before there are any issues tagged with it.

Reply:
I don't think that there will be many potential contributors right now. :wink: